plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -87,13 +90,28 @@ private void createPackageField(Composite container) {
     packageNameLabel.setText(Messages.getString("java.package")); //$NON-NLS-1$
     javaPackageField = new Text(composite, SWT.BORDER);
     
-    ModifyListener pageValidator = new PageValidator();
     javaPackageField.addModifyListener(pageValidator);
 
     GridDataFactory.fillDefaults().grab(true, false).applyTo(javaPackageField);
     GridLayoutFactory.swtDefaults().numColumns(2).applyTo(composite);
   }
 
+  // App Engine service name
+  private void createServiceField(Composite container, ModifyListener pageValidator) {
+    Composite composite = new Composite(container, SWT.NONE);
+    // assumed that container has a single-column GridLayout
+    GridDataFactory.fillDefaults().applyTo(composite);
+
+    Label serviceNameLabel = new Label(composite, SWT.LEAD);
+    serviceNameLabel.setText("App Engine service:");
```

Comment:
I think this would benefit from a tooltip to provide some background on services/modules.

Thinking about this some more, we would benefit from richer tooltips that offers a hyperlink into cloud.google.com for further information.